### PR TITLE
Ref logging

### PR DESCRIPTION
**Pull request by joshLong145**
-fixes to logging across all commands
## Changes

- Item 1
- Item 2

## Checklist

- [ ] Unit tests
- [ ] Documentation

## References *(optional)*

Fixes 
Connects 

<sub> Mention [stepsize] in a comment if you'd like to report some technical debt. See examples [here](https://app.stepsize.com/api/demo-pr-redirect).</sub>

### CODE REVIEW

File: cmd/commands/bucket/query.go [hunk 1]
```diff
@@ -75,7 +74,8 @@ func bootstrapSearchBySchemaIdCommand(ctx context.Context, logger *golog.Logger)
 					continue
 				}
 
-				fmt.Printf("content: %v", itemCnt)
+				logger.Print("content")
+				logger.Print(itemCnt)
```
Comment (mcjcloud) [inline cmd/commands/bucket/query.go:78]: Will this get printed without space between?

File: cmd/commands/registry/Login.go [hunk 2]
```diff
@@ -24,21 +23,21 @@ func bootstrapLoginCommand(ctx context.Context, logger *golog.Logger) (loginCmd
 
 			cli, err := client.New(ctx)
 			if err != nil {
-				logger.Fatalf(status.Error("RPC Client Error: "), err)
+				logger.Fatalf("error while creating rpc client is the daemon running?: %s", err.Error())
```
Comment (mcjcloud) [inline cmd/commands/registry/Login.go:26]: nit: There should be some punctuation between "client is the daemon running"

File: cmd/commands/registry/Login.go [hunk 3]
```diff
@@ -24,21 +23,21 @@ func bootstrapLoginCommand(ctx context.Context, logger *golog.Logger) (loginCmd
...
 			}
 
 			res, err := cli.Login(loginRequest)
 			if err != nil {
-				logger.Fatalf(status.Error("Login Error: "), err)
+				logger.Fatalf("Login Error: ", err.Error())
 				return
 			}
-			logger.Info(status.Debug, "Login Response: ", res)
+			logger.Debug("Login Status: ", res.Success)
```
Comment (mcjcloud) [inline cmd/commands/registry/Login.go:35]: This will print true or false instead of something like "logged in"

**Review by mcjcloud — CHANGES_REQUESTED** [review]

File: cmd/commands/bucket/query.go (hunk 1 above)
Comment (joshLong145) [reply cmd/commands/bucket/query.go:78]: the logger adds a new line

Comment (joshLong145) [reply cmd/commands/bucket/query.go:78]: actually it wont, there is a `println` adding it now

File: cmd/commands/registry/Login.go [hunk 4]
```diff
@@ -23,21 +23,20 @@ func bootstrapLoginCommand(ctx context.Context, logger *golog.Logger) (loginCmd
...
 				return
 			}
-			logger.Debug("Login Status: ", res.Success)
 
 			if res.Success {
-				logger.Info("✅ Login Successful")
+				logger.Print("✅ Login Successful")
 			} else {
-				logger.Fatal("🛑 Login failed")
+				logger.Print("🛑 Login failed")
```
Comment (mcjcloud) [inline cmd/commands/registry/Login.go:39]: These should probably be `Println` as well.

**Review by mcjcloud — APPROVED** [review]

**Pull request merged**